File: 12_reverse_string.js
// reverse a string

function reverseString(str){
    var strArr = str.split("");
    var reverseStrArray = strArr.reverse();
    var reversedString = reverseStrArray.join("");
    return reversedString;

    //return str.split("").reverse().join("");
    
    // another method
    // var final = "";
    // for (var i=str.length-1; i>=0; i--){
    //    final += str[i];
    //}
    // return final;
}

console.log(reverseString("hello"));

File: 12_reverse_string.py
# reverse a string

def reverseString(string):
    strArr = list(reversed(string))
    reverseStrArr = ''.join(strArr)
    return reverseStrArr



def anotherReverseString(string):
    final = ""
    for i in reversed(range(len(string))):
        final += string[i]
    return final


print(reverseString("hello"))
print(anotherReverseString("hello"))